package.json

```diff
@@ -122,20 +124,17 @@
       "<rootDir>/__tests__/utils/",
       "prisma"
     ],
-    "testPathIgnorePatterns": [
-      "<rootDir>/__tests__/utils/"
-    ],
+    "testPathIgnorePatterns": ["<rootDir>/__tests__/utils/"],
     "collectCoverageFrom": [
       "**/*.{ts,tsx}",
       "!**/node_modules/**",
       "!**/*.d.ts",
       "!graphql/(resolvers|typeDefs).ts"
     ]
   },
-  "husky": {
-    "hooks": {
-      "pre-commit": "npm run lint && npm run test && npm run type-check",
-      "pre-push": "npm run lint && npm run test && npm run type-check"
-    }
+  "lint-staged": {
+    "*.{js,jsx,ts,tsx}": "eslint --fix",
+    "*.{json,md,scss}": "prettier --write",
```

Comment:
Does it support `mdx` files formatting?

Reply:
I'm not sure. I'll look into it.

Reply:
There's a mdx plugin for eslint that we can use

https://github.com/mdx-js/eslint-mdx

Maybe you could include in your mdx PR.